build-info/dotnet/core-setup/release/2.0.0/Last_Build_Packages.txt

```diff
@@ -1,45 +1,45 @@
-Microsoft.NETCore.App 2.0.8-servicing-26407-02

```

Comment:
You are moving these versions backwards, do we really need to do that?

Reply:
I was trying to simulate what the auto-PR should have done - I'm fine with not touching the versions in these files if you think that's more correct.

Reply:
Some downstream repo could have already upgraded to the new version and started depending on it, so downgrading is probably not the best.